spec/migrations/20180719163110_add_internal_to_service_template_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_migration
+
+describe AddInternalToServiceTemplate do
+  let(:reserve_stub)          { Spec::Support::MigrationStubs.reserved_stub }
+  let(:service_template_stub) { migration_stub(:ServiceTemplate) }
+
+  migration_context :up do
+    it "migrate reserve data to ServiceTemplate" do
```

Comment:
Can you add a test case without reserved data but with `ServiceTemplateTransformationPlan` type?